synapse/storage/databases/main/stream.py

```diff
@@ -582,6 +582,19 @@ async def get_room_events_max_id(self, room_id: Optional[str] = None) -> str:
             )
             return "t%d-%d" % (topo, token)
 
+    async def get_stream_id_for_event(self, event_id: str) -> int:
```

Comment:
This seems poorly named and bound to be confusing with `get_stream_token_for_event`? A better name would be appreciated!

Reply:
I think it's fair. Stream tokens and stream ids are different things.

tbh I feel like `get_stream_token_for_event` could just be inlined in the two (?) places it is still used. but :man_shrugging:

Reply:
That's fair. I thought keeping it separate was nice to ensure that they both have the same format? Can go either way though.